bundles/core/org.eclipse.smarthome.core.thing.test/src/test/groovy/org/eclipse/smarthome/core/thing/internal/ThingLinkManagerOSGiTest.groovy

```diff
@@ -95,50 +95,50 @@ class ThingLinkManagerOSGiTest extends OSGiTest{
             managedThingProvider.remove(it.getUID())
         }
     }
-    
+
     @Test
     void 'assert that items are linked to thing and channel'() {
         ThingUID thingUID = new ThingUID("hue:lamp:lamp1")
         thingSetupManager.addThing(thingUID, new Configuration(), /* bridge */ null)
-        
+
         Thing thing = thingRegistry.get(thingUID)
         assertThat thing, is(notNullValue())
-        
+
         GroupItem linkedGroupItem = thing.getLinkedItem()
         assertThat linkedGroupItem, is(notNullValue())
         assertThat linkedGroupItem.getName(), is("hue_lamp_lamp1")
-        
+
         def channels = thing.getChannels()
         assertThat channels.size(), is(1)
         Channel channel = channels.first()
-        
+
         def linkedItems = channel.getLinkedItems()
         assertThat linkedItems.size(), is(1)
         Item item = linkedItems.first()
         assertThat item.getName(), is("hue_lamp_lamp1_1")
     }
-    
+
     @Test
     void 'assert that items are unlinked'() {
         ThingUID thingUID = new ThingUID("hue:lamp:lamp1")
         thingSetupManager.addThing(thingUID, new Configuration(), /* bridge */ null)
-        
+
         Thing thing = thingRegistry.get(thingUID)
         assertThat thing, is(notNullValue())
-        
-        thingSetupManager.removeThing(thingUID)
-        
+
+        thingSetupManager.removeThing(thingUID, true)

```

Comment:
No. that is the fixed test, which failed after I have added the doPrivileged call.